array-manipulation-3/src/unique.js

```diff
@@ -0,0 +1,10 @@
+/* eslint-disable no-unused-vars */
+function unique(array) {
+  const result = [];
+  array.filter((x, index) => {
```

Comment:
It's better to use `forEach` if you are not going to use the new array created by `filter`.